README.md

```diff
@@ -84,7 +84,24 @@ docker run --rm -it --net cbio-net \
     migrate_db.py -p /cbioportal/portal.properties -s /cbioportal/db-scripts/src/main/resources/migration.sql
 ```
 
-### Step 5 - Run the cBioPortal web server ###
+### Step 5 - Build Session Service containers
+First, create the mongoDB database:
+
+```
+docker run -d --name=mongoDB --net=cbio-net \
+    -e MONGO_INITDB_DATABASE=session_service \
+    mongo:4.0
+```
+
+Finally, create a container for the Session Service, adding the link to the mongoDB database using `-Dspring.data.mongodb.uri`:
+
+```
+docker run -d --name=cbio-session-service --net=cbio-net -p 8084:8080 \
+    -e JAVA_OPTS="-Dspring.data.mongodb.uri=mongodb://mongoDB:27017/session-service" \
+    thehyve/cbioportal-session-service:docker
```

Comment:
I think the word ‘docker’ is redundant as the version number of a Docker image. Moreover, should we assume that future versions of the session service will be compatible with the current cBioPortal release, or would it be preferable to include an actual version number here?

Reply:
Even in the former case, tagging the image with the default version of ‘latest’ would reduce clutter.

Reply:
This is temporary. 'docker' is a branch name of the repository 'thehyve/cbioportal-session-service'. I have a PR to merge this branch into the 'cBioPortal/session-service' master branch. Once this is done, then this name can be replaced.

Reply:
That explains the `thehyve/` part, but my question was about the ‘:docker’ version number. Will that tag remain compatible with cBioPortal v2.1.0 for as long as we'll have a reason to maintain installations of cBioPortal v2.1.0?

Reply:
I see that you changed the tag to specifically mention cbiov2.1.0. That should be good and keep things reproducible 👍